packages/web-components/src/components/masthead/__stories__/cloud-masthead.stories.ts

```diff
@@ -30,6 +30,21 @@ const platformData = {
   url: 'https://www.ibm.com/cloud',
 };
 
+const platformObject = {
+  'en-US': {
+    url: 'https://www.cloud.com/us-en',
+    name: 'Cloud',
```

Comment:
I don't think we need to account for `platformObject.local.name` in the object as from what I understand Product Name doesn't go through the globalization process.

Reply:
@proeung, `name` is optional in `platformObj`. If you want to show the same platform name for all locales with different URLs, you can combine `platformObj` with the `platform` attribute

So the following object would would work as long as `platform` exists in `dds-cloud-masthead-container`:

```
const platformObject = {
  'en-US': {
    url: 'https://www.cloud.com/us-en',
  },
  'fr-FR': {
    url: 'https://www.cloud.com/fr-fr/sample',
  },
  'es-MX': {
    url: 'https://www.cloud.com/ibm/sample/es-mx',
  },
};
```

If this is too confusing, I can remove name completely.

Reply:
Name has been removed 👍